appcues/src/main/java/com/appcues/ui/ExperienceRenderer.kt

```diff
@@ -101,25 +110,40 @@ internal class ExperienceRenderer(
     }
 
     suspend fun show(experiences: List<Experience>) {
-        showEmbeds(experiences.filter { it.renderContext is RenderContext.Embed })
-        showModal(experiences.filter { it.renderContext is RenderContext.Modal })
+        // get any experiences if exist, and check if qualification trigger is screen_view
+        experiences.firstOrNull()?.let {
+            if (it.trigger is Qualification && it.trigger.reason == "screen_view") {
+                // clear list in case this was a screen_view qualification
+                potentiallyRenderableExperiences.clear()
+            }
+        }
```

Comment:
following along - so just looking at the first experience here should be good enough, since all in a qualification batch would have the same trigger reason, i think that works.

Reply:
would there be any other to check this?

Reply:
No, I think this works. Technically the `reason` is at the root of the response - and then an array of experiences are provided. But we do copy that `reason` to every experience I think during mapping - so this works.